packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -66,6 +67,10 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
   const [menuHidden, setMenuHidden] = useState<boolean>(true);
   const containerRef = useRef<HTMLDivElement>(null);
   const theme = useTheme();
+  const { strings } = useLocale();
```

Comment:
I think we want to take in the "locale strings" as just a string prop. It should be in the composite code to hook in the strings from the localization provider as the correct prop to be used in a component.

Reply:
+1 to this, we need to keep components completely pure (they should never rely on being wrapped in a provider).
I think we'll need to create a task to remove the `useTheme` from components also (getTheme should hopefully work instead)

Reply:
Filed a bug for the useTheme (https://skype.visualstudio.com/SPOOL/_workitems/edit/2495786). Resolving this.